diff --git a/src/fennel/macros.fnl b/src/fennel/macros.fnl
index 9788120..f8231f8 100644
--- a/src/fennel/macros.fnl
+++ b/src/fennel/macros.fnl
@@ -1,5 +1,8 @@
 ;; fennel-ls: macro-file
 
+;; Global helpers exposed on built-in macros' compiler-env from src/fennel.fnl:
+;; * {: fennel : utils :get-function-metadata specials.get-function-metadata}
+
 ;; These macros are awkward because their definition cannot rely on the any
 ;; built-in macros, only special forms. (no when, no icollect, etc)
 
